packages/nextjs/src/config/webpack.ts

```diff
@@ -764,7 +764,7 @@ export function getWebpackPluginOptions(
     project: process.env.SENTRY_PROJECT,
     authToken: process.env.SENTRY_AUTH_TOKEN,
     configFile: hasSentryProperties ? 'sentry.properties' : undefined,
-    stripPrefix: ['webpack://_N_E/'],
+    stripPrefix: ['webpack://_N_E/', 'webpack://'],
```

Comment:
Does this really help?
Because in our example, this would just turn `webpack://sentry-docs/app/changelog/%5Fadmin/page.tsx` into `sentry-docs/app/changelog/%5Fadmin/page.tsx`

not quite sure if thats better?